No more hard coding Native Libraries

jbonnett92 opened the pull request:
The new class changes the native library depending on what OS the program is ran from.

jbonnett92 committed:
Update Main.java

Added Library loader, to load a particular library depending on what OS the program is running on

jbonnett92 committed:
Create LibraryLoader.java

Library loader class, to load a particular library depending on the OS running the program.
This means you do not have to set the native library location for the lwjgl.jar.

fhntv24 commented:
If you want it to be accepted - better to move LibaryLoader to engine.core
oh and btw it will not work , beacos OpenAL isnt used here at all.
And folders are lib/natives/someting .Oh , and how about "solaris"? He needs diffrent libs from linux i think.

jbonnett92 commented:
It work's with mine... Tried on all 3.

jbonnett92 committed:
Update LibraryLoader.java

fhntv24 commented:
Solaris folder there is for reason.You know that,right? 

Oh and really - its better to be in engine.core , its more logical to be there..
And maybe you will make libs.jar loader as well?

jbonnett92 committed:
Update LibraryLoader.java

jbonnett92 committed:
Create LibraryLoader.java

Moved from com.base.game...
Allows you to automatically load native libraries rather than hard coding them into "lwjgl.jar" via native library location.

jbonnett92 committed:
Delete LibraryLoader.java

jbonnett92 commented:
Well I did the changes apart from creating the libs.jar.

fhntv24 commented:
Oh you cold just edit path when editing file , its easy.Just hit backspace on begining of filename.Its better that creating/deleting files ;)
